staging/src/k8s.io/kubectl/pkg/cmd/config/set_credentials.go

```diff
@@ -306,6 +317,10 @@ func (o *setCredentialsOptions) modifyAuthInfo(existingAuthInfo clientcmdapi.Aut
 			modifiedAuthInfo.Exec.Args = o.execArgs
 		}
 
+		if o.execInteractiveMode.Provided() {
+			modifiedAuthInfo.Exec.InteractiveMode = clientcmdapi.ExecInteractiveMode(o.execInteractiveMode.Value())
+		}
+
```

Comment:
why not set ProvideClusterInfo here?

Reply:
Thanks for review. Moved this `modifiedAuthInfo.Exec.ProvideClusterInfo = o.execProvideClusterInfo` in here.